docs/source/conf.py

```diff
@@ -277,6 +277,7 @@
     "torch.quantization",
     "torch.quantization.fx",
     "torch.sparse",
+    "torch.nested",
```

Comment:
We should not add undocumented public namespaces please.
As mentioned in the comment above. You should NOT add anything new to this list!

Reply:
There is documentation here actually, checkout the preview. https://docs-preview.pytorch.org/72881/index.html

Reply:
Perfect, so this line can be removed :)

Reply:
Thanks for the update! :)